collections/business/general-notifications/event-general-notifications/pp-ach-upcoming-cut-off-time/event-general-notifications.json

```diff
@@ -0,0 +1,34 @@
+{
+  "eventClassName": "com.backbase.positivepay.event.spec.v1.PpAchUpcomingCutOffTimeEvent",
+  "id": "pp-ach-upcoming-cut-off-time",
+  "eventBusinessName": "Positive Pay ACH Upcoming cut-off time",
+  "recipientType": "subscription",
+  "extractor": "#this['arrangementId']",
+  "subscriptionType": "arrangement",
```

Comment:
@bartlomiejo-bb Does this event also have service agreement id in payload?

Reply:
No, only `arrangementId` and `cutOffTime`